sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample4_MultivariateDetect.cs

```diff
@@ -140,17 +137,15 @@ private async Task<DetectionResult> detectAsync(AnomalyDetectorClient client, st
                 Guid result_id = Guid.Parse(result_id_path.Split('/').LastOrDefault());
                 // get detection result
                 Response<DetectionResult> result = await client.GetDetectionResultAsync(result_id).ConfigureAwait(false);
-                int tryout_count = 0;
-                while (result.Value.Summary.Status != DetectionStatus.Ready & tryout_count < max_tryout)
+                while (result.Value.Summary.Status != DetectionStatus.Ready & result.Value.Summary.Status != DetectionStatus.Failed)
                 {
                     System.Threading.Thread.Sleep(2000);
                     result = await client.GetDetectionResultAsync(result_id).ConfigureAwait(false);
-                    tryout_count += 1;
                 }
 
                 if (result.Value.Summary.Status != DetectionStatus.Ready)
                 {
-                    Console.WriteLine(String.Format("Request timeout after {0} tryouts", max_tryout));
+                    Console.WriteLine(String.Format("Request timeout after {0} tryouts"));
```

Comment:
Impropriate console output. Make it like "Inference failed". And print the errors in the result if necessary.